dev/docker/hive/Dockerfile

```diff
@@ -9,6 +9,7 @@ LABEL maintainer="support@datastrato.com"
 ARG HADOOP_PACKAGE_NAME
 ARG HIVE_PACKAGE_NAME
 ARG JDBC_DIVER_PACKAGE_NAME
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Where use this `DEBIAN_FRONTEND` arg? I didn't find it anywhere.

Reply:
I added it because I moved 'mysql-sever' to basic tools.

Reply:
OK, I found `DEBIAN_FRONTEND` it in the
https://askubuntu.com/questions/972516/debian-frontend-environment-variable